### ISSUE TITLE
Sample

### ISSUE THREAD

**Issue by scls19fr** [coursera/pandas-ply#7]
Hello,

I'm trying your package but it will be nice to improve doc to tell where to find `flights` sample dataframe.
I have been looking inside `dplyr` package http://cran.r-project.org/web/packages/dplyr/index.html
but I wasn't able to find it.
Thanks

Kind regards

**Comment by jkeirstead** [comment]
There's an example of how to get this data from the web using Python at https://plot.ly/python/widget-app/#nyc-flights-database

**Comment by s-celles** [comment]
So it's using

``` python
df = pd.read_csv('https://raw.githubusercontent.com/yankev/testing/master/datasets/nycflights.csv')
df = df.drop(df.columns[[0]], axis=1) 
```

Thanks

**Comment by jkeirstead** [comment]
Exactly, though of course the data will be in an object called `df` not `flights` but that's easy enough to fix.
